configure.ac

```diff
@@ -137,7 +137,7 @@ AC_MSG_CHECKING([for jar installation path])
 AC_ARG_WITH(jardir,
             AS_HELP_STRING([--with-jardir],
                            [jar file installation path [LIBDIR/jni]]),
-            jardir=${withval}, jardir=${libdir}/jni)
+            jardir=${withval}, jardir=${prefix}/lib/jni)
```

Comment:
Cerbero automatically passes --libdir=${prefix}/lib and if you build using just autotools, $exec_prefix is set to $prefix, so it should be all fine (in theory). What exactly went wrong in your case? 

The problem with using ${prefix}/lib instead of ${libdir} is that the lib sub-dir within ${prefix} can also be lib64 etc (see self.config.lib_suffix), so hard-coding it inside configure.ac bad.